docs/Changelog.md

```diff
@@ -61,6 +61,8 @@ and `GenericNodeTransformation.implementation` is not invoked in this case
 * [#2474](https://github.com/TouK/nussknacker/pull/2474) Interpreter return type changed from `F[Either[List[InterpretationResult], EspExceptionInfo[_ <: Throwable]]]` to `F[List[Either[InterpretationResult, EspExceptionInfo[_ <: Throwable]]]]`.
   Hence, e.g. multiple branches in Graph can be evaluated, both positively and negatively at the same time.
 * [#2441](https://github.com/TouK/nussknacker/pull/2441) avro sink supports defaults of primitive avro types
+* [#2498](https://github.com/TouK/nussknacker/pull/2498), [#2499](https://github.com/TouK/nussknacker/pull/2499), [#2503](https://github.com/TouK/nussknacker/pull/2503), [#2539](https://github.com/TouK/nussknacker/pull/2539) EspExceptionHandler is removed from ProcessConfigCreator. 
```

Comment:
Let's move it to 1.2.0 :) also - probably former changes?

Reply:
:+1:

Reply:
done: https://github.com/TouK/nussknacker/pull/2539/commits/40365223a88fbfb5bdb2525f485cd3fc79922069